test/fixtures/dynamodb/dynamodb_test.go

```diff
@@ -337,29 +317,66 @@ var ddbSettings = &ddb.Settings{
 	},
 }
 
-func (s *DynamoDbSuite) dynamoDbDisabledPurgeFixtures() []*fixtures.FixtureSet {
-	return []*fixtures.FixtureSet{
-		{
-			Enabled: true,
-			Writer:  fixtures.DynamoDbFixtureWriterFactory(ddbSettings),
-			Fixtures: []interface{}{
-				&Person{Name: "Ash", Age: 10},
-			},
+func (s *DynamoDbSuite) dynamoDbFixtures1(ctx context.Context, config cfg.Config, logger log.Logger) (fixtures.NamedFixtures[*Person], fixtures.FixtureWriter, bool, error) {
```

Comment:
these could also just return the FixtureSet already?
Could it even look something like this?
```
func (s *DynamoDbSuite) dynamoDbFixtures(purge bool, data fixtures.NamedFixtures[*Person]) ([]fixtures.FixtureSet, error) {
	writer, err := fixtures.NewDynamoDbFixtureWriter(s.Env().Context(), s.Env().Config(), s.Env().Logger(), ddbSettings)
	if err != nil {
		return nil, fmt.Errorf("failed to create ddb fixture writer: %w", err)
	}
	
	return []fixtures.FixtureSet{fixtures.NewSimpleFixtureSet(data, writer, true, purge)}, nil
}
```

Reply:
Pls check my implementation. Do you agree with it?